src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/GeneralType.php

```diff
@@ -46,25 +46,78 @@ class GeneralType extends TranslatorAwareType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('catalog_mode', SwitchType::class)
-            ->add('catalog_mode_with_prices', SwitchType::class)
+            ->add('catalog_mode', SwitchType::class, [
+                'label' => $this->trans('Catalog mode', 'Admin.Shopparameters.Feature'),
+                'help' => $this->trans(
+                        'Catalog mode disables the shopping cart on your store. Visitors will be able to browse your products catalog, but not buy them.',
+                        'Admin.Shopparameters.Help'
+                    ) . '<br />' . $this->trans(
+                        'Have specific needs? Edit particular groups to let them see prices or not.',
```

Comment:
This help text could be clearer on where to perform the action.

```suggestion
                        'To hide prices for a specific group, go to [1]Customer Settings > Groups[\1].',                        
```

Also, this help text should be placed under the "Show prices" field instead of "Catalog mode".